packages/cf-style-provider/package.json

```diff
@@ -31,4 +31,4 @@
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0-0 || ^16.0.0-0"
   }
-}
+}
```

Comment:
No worries, I really like VSCODE but if sublime is your jam you should just stick with what you like :)